src/LLT.cpp

```diff
@@ -17,8 +19,24 @@ PYBIND11_MODULE(_eigenLLT, m) {
   m.def(
       "computeAndSolve",
       [](Eigen::MatrixXf a, Eigen::MatrixXf b) {
+        std::cout << "Here is the matrix A:\n" << a << std::endl;
```

Comment:
Do we need to have all these comments in here?

Reply:
The data gets thrown out before the return line, so it just returned junk data. The comments are printed so that the not-junk data gets seen. I could just return nothing looking at the code, now, since the prints take care of the solution, or I could modify it so that the solution gets stored

Reply:
I mean, the purpose of this function is to find a solution, and the verbosity could help with that

Reply:
looking further, the functions are originally used as voids with outputs being references to pointers, so I may just remove the return statements

Reply:
I think it would be good to not have a bunch of print statements when using a function so as to not clutter the terminal. If you think otherwise just mark this as resolved and I'll merge.